test/test_cli.py

```diff
@@ -17,24 +17,24 @@ def mkdtemp():
     shutil.rmtree(_directory)
 
 
-def test_makedirs_do_not_overwrite():
-    _directory, _ = mkdtemp().next()
+def test_makedirs_do_not_overwrite(mkdtemp):
+    _directory, _stat = mkdtemp
     with pytest.raises(OSError):
-        makedirs(_directory)
+            makedirs(_directory)
```

Comment:
Why the increase in the indent? Seems wrong.

Reply:
Nice catch! I used a context manager and them remove it but the indentation remained. Fixing it now...

Reply:
Great

Reply:
BTW, kind of related, I started using `black` which became the "the facto" code formatter for python:

https://black.readthedocs.io/en/stable/

It solves the discussion, checking, review of code style for good. Just run black before committing and never worry about code format ever again.

For existing projects applying it is a huge commit and should be done only if all the contributors are on board first. (B/c of the conflicts it will create.)